pipelines/New-pipeline/pipeline-tutorial-template.md

```diff
@@ -0,0 +1,46 @@
+# Title of your pipeline
+### Author: Author name
+#### Reviewed by: Reviewer names
+
+*This is a template for creating a tutorial for a pipeline, which will be included in the online documentation. These are the suggested sections, but the contributor can include additional sections or change titles if desired.* 
+
+## Introduction
+*Write an introduction here that is a brief description of what the pipeline does, and the context in which it should be used. This should include any relevant citations to the original paper.*
+
+## Before you start
+*Write any necessary data, API keys, or anything else that the user may need before they run the pipeline.*
+
+## Pipeline steps
+*Explain the pipeline steps (i.e. scripts) here so the user understands what each step is doing. This can include diagrams or screenshots if it will help the user. This should include information about how to understand the results.*
+
+For example:
+### Pipeline inputs
+This is a description of the inputs of the pipeline and what they mean.
+
+### 1. Retrieving the data
+This step retrieves data from this place using this API. 
+
+### 2. Cleaning the data
+This step uses this package to clean the data for this reason. 
+
+### 3. Analyzing the data
+This step analyzes this data to produce this indicator. This is how it works. 
+
+Etc.
+
+### Pipeline outputs
+This is a description of the outputs of the pipeline and what they mean. 
+
+## Example
+*Include an example of a pipeline run here. If possible, this should be the same as the example that autofills in the pipeline so the user can easily run it and see the results. Include images of the outputs here so that the user can get an idea of what it they should look like.*
+
+## Pipeline limitations
+*Describe any limitations to the pipeline that should be communicated.*
```

Comment:
I would put the limitations higher up, after the introduction. 

Basically : 
intro = this is what it's for...
limitation = but be careful...